float32 forward_left
float32 forward_right
float32 top_left
float32 top_right
float32 max_level
bool  stop

